posts/inside-rust/2022-04-13-imposter-syndrome.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Imposter Syndrome"
+author: Jane Lusby Project Director of Collaboration
+team: Rust Foundation Project Directors <https://foundation.rust-lang.org/about/>
+---
+
+*Preface: This is in response to some feedback the project directors received
+from the Rust Foundation staff. Some of the contributors they'd talked to said
+they didn't feel justified in applying for Foundation grants even though they'd
+love the opportunity, because they don't feel qualified or deserving of them
+compared to the other amazing contributors they look up to within the Rust
+project. This was a little bit heart breaking to me, because I know exactly
+what that feeling is like[^1], and also know just how wrong they are.*
+
+Imposter syndrome is an insidious problem within software communities. Many of
+us, especially members of marginalized communities, struggle to shake the
+feeling that we aren't as qualified as our peers. This makes us feel
+unqualified and undeserving compared to those around us. It can make us
+hesitate to join communities in the first place. For those already involved, it
+can create a sense of impending doom where you constantly feel like you're
+going to get found out and expelled from the community. It overall just not
+great for mental health, 0/10 would not recommend.
```

Comment:
This confused me at first as I read as a double negative of sorts wherein zero out of ten wouldn't recommend (since that could mean ten would?).

Would a comment help clarify or have I simply misunderstood?

```suggestion
great for mental health, 0/10, would not recommend.
```